en/docs/observe/api-manager/monitoring-correlation-logs.md

```diff
@@ -13,7 +13,11 @@ Furthermore, when observability is enabled in WSO2 API-M, a random Correlation I
 
 ## Enabling Correlation Logs
 
-Enabling observability is simple in API Manager. All you need to do is to find the following system property in the product startup script (stored in the `<API-M_HOME>/bin/` directory) and set it to `true`. By default, this is set to `false`.
+Correlation logs are disabled by default and it can be enabled at the server startup using the system parameter OR during the runtime using the Devops REST API. 
```

Comment:
```suggestion
Correlation logs are disabled by default and it can be enabled at the server startup using the system parameter OR during the runtime using the DevOps REST API. 
```